src/NzbDrone.Core/Indexers/Rarbg/RarbgRequestGenerator.cs

```diff
@@ -10,6 +10,14 @@ public class RarbgRequestGenerator : IIndexerRequestGenerator
     {
         private readonly IRarbgTokenProvider _tokenProvider;
 
+        private string _categoryParam
+        {
+            get
+            {
+                return string.Join(";", Settings.Categories);
```

Comment:
Make this `return string.Join(";", Settings.Categories.Distinct());` to avoid passing same category twice if user enters it.

Also you could likely just get rid of this section and use `string.Join(";", Settings.Categories.Distinct())` below... but looks fine either way. 

I'm going to go ahead and pull this into V2 aphrodite branch as well.

Reply:
It has been done. I removed the block